src/test/mochitest/browser_dbg-pretty-print.js

```diff
@@ -3,29 +3,31 @@
 
 // Tests basic pretty-printing functionality.
 
-add_task(function* () {
-  const dbg = yield initDebugger("doc-minified.html");
+async function prettyPrintTest() {
+  const dbg = await initDebugger("doc-minified.html");
```

Comment:
Is it possible to keep add_task/yield?
That would allow you to not have to refactor every single mochitest?
Also wouldn't it be possible to not reimplement all the deps from head/shared-head?
I would be happy to review m-c patch to help you reuse some existing code.